ld: duplicate symbol _h80211 in ./.libs/libaircrack.a(libaircrack_la-communications.o) and src/airodump-ng/airodump_ng-airodump-ng.o for architecture x86_64

--- src/airodump-ng/airodump-ng.c.orig	2022-04-27 22:45:13.000000000 -0400
+++ src/airodump-ng/airodump-ng.c	2022-04-27 22:46:01.000000000 -0400
@@ -92,8 +92,6 @@
 #include "radiotap/radiotap_iter.h"
 
 struct devices dev;
-uint8_t h80211[4096] __attribute__((aligned(16)));
-uint8_t tmpbuf[4096] __attribute__((aligned(16)));
 
 static const unsigned char llcnull[] = {0, 0, 0, 0};
 
